### ISSUE TITLE
Infinite Load (Custom Skin Pro) loads pages and other posts

### ISSUE THREAD

**Issue by sebastiantheobald** [dudaster/ele-custom-skin#259]
Hello everyone, 

I was very excited about the new feature "Infinite Load". I enabled it within my posts widget on this site: 
https://cavalearn.com/courses/

When loading the next posts the Infinite Loader loads the completely Homepage, which crashes the whole page.  

I set Post Type "Courses" within the query options and set Query ID to "jet-smart-filters". This is because of using the jet smart Filters which I use one the site. 

Does anybody know this issue and could help?

**Comment by dudaster** [comment]
yeah. it is wierd. Can you switch it to load more to have more control to see what is going on there?

**Comment by sebastiantheobald** [comment]
Already did that.. same like before.. But I will reactivate it for you.

**Comment by dudaster** [comment]
It seems to be a redirection issue there.

**Comment by sebastiantheobald** [comment]
Redirection for what?

**Comment by sebastiantheobald** [comment]
The issue is in Firefox and Safari, but not in Chrome.. weird.

**Comment by dudaster** [comment]
i'm getting the same issue on chrome. Disable and clean all cache and try again.

**Comment by sebastiantheobald** [comment]
okay, and do you have a idea whats the problem?

**Comment by dudaster** [comment]
I could not tell it is a server side issue. Do you have chaching plugins?

**Comment by sebastiantheobald** [comment]
No, not yet.. 
I dont understand whats happening.. The button should load the next posts of the defined post type and it loads the homepage. Seems to be a basic problem

**Comment by sebastiantheobald** [comment]
Hello again :) Is their a possibility to solve this issue?

**Comment by dudaster** [comment]
is there a way to set up a staging site?

**Comment by betzebu** [comment]
We are already working on the page. Is not live yet. So if you want to test something, thats possible

**Comment by dudaster** [comment]
please go on the chat on dudaster.com
